app/src/main/res/layout/profile_reset_pin_activity.xml

```diff
@@ -52,7 +52,6 @@
         android:layout_height="wrap_content"
         android:overScrollMode="never"
         android:scrollbars="none"
-        app:layout_constraintEnd_toEndOf="parent"
```

Comment:
Other constraints need to be removed as well.

Reply:
done